compliance/virtualmachines/agent/config/config.go

```diff
@@ -0,0 +1,13 @@
+package config
```

Comment:
Not sure why we have the separate package for the config. Wouldn't `agent.Config` read better than `config.AgentConfig`?